### PR TITLE
Fix/us0003/zero column added  in lz config

### PR DESCRIPTION
**Pull request by gabmota8**
fixed a bug in the lz-config , now we can added columns with zero index

### CODE REVIEW

File: web-client/src/components/lz-config/LZColumnSection.vue [hunk 1]
```diff
@@ -31,10 +31,12 @@ const toAdd = defineModel('toAdd', {
 	},
 })
 const addColumn = () => {
-	if(!toAdd.value || toAdd.value === '') return
+	console.log("clicou")
```
Comment (pauloarantesmachado) [inline web-client/src/components/lz-config/LZColumnSection.vue:34]: Please, remove the console.log

File: web-client/src/components/lz-config/LZColumnSection.vue [hunk 2]
```diff
@@ -31,10 +31,12 @@ const toAdd = defineModel('toAdd', {
...
+	if (toAdd.value === undefined || toAdd.value === '') return // Verifica se é indefinido ou vazio
 	const columnIndex = toAdd.value as number
 	columnList[columnIndex].status = 1
 	setActiveCount()
+	console.log("cadastrou!!")
```
Comment (pauloarantesmachado) [inline web-client/src/components/lz-config/LZColumnSection.vue:39]: Please, remove the console.log

File: web-server/src/main/java/com/domrock/configurator/Model/ConfigModel/ColumnConfig.java
Comment (pauloarantesmachado) [inline web-server/src/main/java/com/domrock/configurator/Model/ConfigModel/ColumnConfig.java:1]: I don't think it needs to go up, as it's something for another task

File: web-client/src/components/lz-config/LZColumnSection.vue (hunk 2 above)
Comment (gabmota8) [reply web-client/src/components/lz-config/LZColumnSection.vue:39]: done!!

File: web-client/src/components/lz-config/LZColumnSection.vue (hunk 1 above)
Comment (gabmota8) [reply web-client/src/components/lz-config/LZColumnSection.vue:34]: done

**Pull request merged**